definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-resp-sol-complete-aos-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
@johnoneill46 do we need this new flag?

I think we don't, but I might be wrong. Please confirm with Jeremy.

Feature toggles flag are for release. If epic is going to be released as "RRJ" - please use RRJ. Otherwise use specific flag.

Reply:
My understanding was that the PostFix was to do with the Epics on the stories. From that we could release multiple epic files at once or independantly. 
But seemingly i was wrong and that is not the case. Is there a place that i can reference that says what epics belong in what postfix?

Ill update my PR accordingly